doc/whats_new.rst

```diff
@@ -53,6 +53,9 @@ Changelog
 
     - Add new class :class:`mne.decoding.EMS` to transform epochs with the event-matched spatial filters and add 'cv' parameter to :func:`mne.decoding.compute_ems`, by `Jean-Remi King`_
 
+    - Added :class:`mne.time_frequency.EpochsTFR` and average parameter in :func:`mne.time_frequency.tfr_morlet` and :func:`mne.time_frequency.tfr_multitaper` to compute time frequency transforms on epochs without averaging, by `Jean-Remi King`_ and `Alex Gramfort`_
```

Comment:
'to compute time frequency transforms on single trial epochs' better to define by positive than negative aspects